dracut/30ignition/coreos-teardown-initramfs-network.sh

```diff
@@ -4,6 +4,10 @@
 
 set -euo pipefail
 
+# Use tmpfiles.d to schedule SELinux relabel of files
+schedule_selinux_relabel() {
+    echo "Z $1 - - -" >> "/run/tmpfiles.d/$(basename $0)-relabel.conf"
```

Comment:
Can we use https://github.com/coreos/fedora-coreos-config/blob/testing-devel/overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/coreos-relabel instead so we relabel from the initrd on FCOS?

It's awkward since it's in a different repo. But really everything in this repo that's not strictly about Ignition should be moved to overlays eventually and then this repo merged into https://github.com/coreos/ignition.

(And eventually RHEL8 will also get the patch for relabeling from the initrd, so it'll do the better thing on RHCOS too.)

Reply:
I originally was using `coreos-relabel` but I didn't want to introduce another dep here in this repo. I agree we should eventually move this into an overlay or something because it's really blurring the line between what should be in `ignition-dracut` vs in an overlay.

Maybe we should leave as and if we move it to the overlay switch it to `coreos-relabel`?

Reply:
Sure, though... a lot of work went into not having to do these hacks anymore. :cry:

How about just in this function do something like `command -v coreos-relabel` and then fallback to what you're doing here?

Reply:
done!